docs/ci/ci-remove-dco-check.md

```diff
@@ -0,0 +1,11 @@
+# Remove DCO Sign-off Check
+
+## Overview
+
+Presently, all repositories under our `instructlab` GitHub org require contributors to add a [Developer Certificate Origin (DCO)](https://developercertificate.org/) sign-off in each commit to indicate that they agree to the DCO legal terms. While these DCO sign-offs can be a great way for us to validate that a contributor agrees to the DCO legal terms, many upstream projects actively use the DCO without explicitly requiring contributors to add "Signed-off-by" on their commits. So in other words, we can still require contributors to agree to the DCO terms, but we don't necessarily _need_ contributors to add DCO sign-offs in each commit.
+
+## Changes Going Forward
+
+We should add DCO text at the top level of each InstructLab repository and update each repository's `CONTRIBUTING.md` guide to clearly state that anyone who makes a contribution to InstructLab will be implicitly agreeing to the DCO for that particular repository.
```

Comment:
What does "top level of each repository" mean?

Reply:
Yep, I should clarify. 

What "top level of each repository" means is: we should put the physical DCO text in a file in the repository.

Here are examples of a few sizeable repos that are doing this today:
- https://github.com/tracim/tracim/blob/35466d53b257e1ea24538d4e163b37266c02ecd4/docs/DCO
- https://github.com/gnuradio/volk/blob/main/docs/DCO.txt

However, in our case, we want our `DCO.txt` file to be at the root of the repository -- at the same level as the `README.md`.

I will update the wording to be more clear about that!

Reply:
Updated. Let me know what you think